gateway-service-impl/src/main/java/org/hypertrace/gateway/service/entity/EntitiesRequestContext.java

```diff
@@ -28,6 +31,22 @@ public String getTimestampAttributeId() {
     return timestampAttributeId;
   }
 
+  public boolean canApplyLimit() {
+    return applyLimit;
+  }
+
+  public void canApplyLimit(boolean applyLimit) {
```

Comment:
nit: these method names are a bit confusing. The fact that one is a get and another is a set is not clear.

Reply:
got rid of the methods. no longer needed as discussed